meshtastic/module_config.proto

```diff
@@ -558,3 +568,40 @@ message ModuleConfig {
 
   }
 }
+
+/*
+ * A GPIO pin definition for remote hardware module
+ */
+ message RemoteHardwarePin {
+  /*
+   * GPIO Pin number (must match Arduino)
+   */
+  uint32 gpio_pin = 1;
```

Comment:
especially for the repeated type, you can limit the uint32 in options to 8 bits...

*RemoteHardwarePin.gpio_pin int_size:8